provision-team/provision_aks.sh

```diff
@@ -97,7 +115,7 @@ kvstore set ${teamName} ACR_URI $ACR_ID
 echo "Granting Service Princpal " $SP_NAME " access to ACR " $teamName"acr" "..."
 (
     set -x
-    az role assignment create --assignee $SP_ID --role Reader --scope $ACR_ID
+    az role assignment create --assignee $SP_ID --role acrpull --scope $ACR_ID
```

Comment:
Is this correct?  https://docs.microsoft.com/en-us/azure/container-registry/container-registry-auth-aci#create-a-service-principal The doc seems to indicate this can be set during Service Principal creation.  Not sure what is best method.

Reply:
It can, but there are bugs with AKS + ACR auth so this is why we did it this way.
https://github.com/Azure/AKS/issues?utf8=%E2%9C%93&q=is:issue+is:open+ACR